client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/ContactForm/ContactForm.tsx

```diff
@@ -81,73 +81,77 @@ const ContactForm: React.FC<Props> = (props: Props): JSX.Element => {
-                                },
-                                InteractionEventContactFields.PHONE_NUMBER
-                            )}
+                                              required={firstName.length > 0}
+                                              className={classes.newContactField}
+                                              value={phoneNumber.number}
+                                              isValid={phoneNumber.isValid}
+                                              setIsValid={isValid => onChange(
+                                                  {
+                                                      ...phoneNumber,
+                                                      isValid: isValid
+                                                  },
+                                                  InteractionEventContactFields.PHONE_NUMBER
+                                              )
+                                              }
+                                              onChange={event => onChange(
+                                                  {
+                                                      ...phoneNumber,
+                                                      number: event.target.value,
+                                                  },
+                                                  InteractionEventContactFields.PHONE_NUMBER
+                                              )}
                         />
                     </FormInput>
                 </Grid>
             </Grid>
             <Grid className={formClasses.formRow} container justify='flex-start'>
-                <Grid item xs={4}>
+                <Grid item xs={4} className={classes.contactAdditionalDetails}>
                     <FormInput fieldName={contactedPersonID}>
                         <AlphanumericTextField
+                            className={classes.newContactField}
                             errors={errors}
                             setError={setError}
                             clearErrors={clearErrors}
                             name={InteractionEventContactFields.ID}
-                            className={classes.newContactField}
                             value={id}
                             onChange={newValue => onChange(newValue, InteractionEventContactFields.ID)}
                         />
                     </FormInput>
                 </Grid>
-                <Grid item xs={4}>
+                <Grid item xs={4} className={classes.contactAdditionalDetails}>
                     <FormInput fieldName={contactTypeField}>
                         <FormControl fullWidth>
                             <InputLabel>סוג מגע</InputLabel>
                             <Select
                                 test-id={'contactType'}
+                                className={classes.newContactField}
                                 label='סוג מגע'
```

Comment:
![image](https://user-images.githubusercontent.com/42100027/94041485-f4307f80-fdd2-11ea-93d4-dd31a373a983.png)
label is in weird position